src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  min-width: 1024px;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
+  margin: auto;
+  margin-left: 50px;
+}
```

Comment:
The 'logo' class has margins set individually. It's better to use shorthand notation for margins. For example, instead of setting `margin: auto;` and `margin-left: 50px;` separately, you can write `margin: auto auto auto 50px;`.